chain/store/name.go

```diff
@@ -80,7 +195,7 @@ func (sdb *StateDB) getName(registrant []byte) (string, error) {
 
 	if len(enc) > 0 {
 		name = string(enc)
-		nameId := getNameId(name)
+		nameId := "test"
```

Comment:
We no longer need this `getName` method in the new name service because there will be no more pk -> name map stored in db.